packages/vm/tests/api/state/stateManager.spec.ts

```diff
@@ -514,6 +513,31 @@ tape('StateManager - Contract storage', (tester) => {
     t.ok(contractValue.equals(expect), 'trailing zeros are not stripped')
     t.end()
   })
+
+  it('should get and verify EIP 1178 proofs', async (t) => {
+    const address = Address.zero()
+    const key = zeros(32)
+    const value = Buffer.from('0000aabb00', 'hex')
+    const code = Buffer.from('6000', 'hex')
+    const stateManager = new DefaultStateManager()
+    await stateManager.checkpoint()
```

Comment:
Note: without checkpoint/commit the cache is not flushed which means that the MPT cannot find the right path for the requested account (this should be addressed, ensure that cache is flushed before trying to create a proof)

Reply:
I guess you can just call into `cache.flush()` in the new proof functions? 🤔

Reply:
Ah yes that should work and makes more sense.

Reply:
in normal vm operation the stateManager should checkpoint and commit after every block, so we should never run into a scenario where this occurs, correct? moreso just a spec test setup particularity? just wanted to make sure before i resolved this conversation